README.md

```diff
@@ -64,3 +64,4 @@ A simple tic tac toe game written in plain vanilla Java.
 - Store state 
 - Tests
 
+edit
```

Comment:
That is awesome. Could make the following changes

Reply:
@yeluochangkong make this private

Reply:
+1

Reply:
+2

Reply:
+3